brainscore_vision/__init__.py

```diff
@@ -30,6 +33,10 @@ def load_dataset(identifier: str) -> Union[DataAssembly, Any]:
     return data_registry[identifier]()
 
 
+def load_stimulus_set(identifier: str) -> Any:
+    return stimulus_set_registry[identifier]()
```

Comment:
there's an `import_plugin` missing here I think -- @kvfairchild, can we easily add this here or is there something to consider?

Reply:
As long as there's a `stimulus_set` directory structured like any other plugin directory, it should be fine to just add it.

Reply:
@samwinebrake I will take care of this

Reply:
@kvfairchild am I missing something or is this as simple as adding `import_plugin('brainscore_vision', 'data', identifier)` (notice **data** instead of **stimulus_set**)

Reply:
Not quite that simple; `locate_plugin()` in `core/brainscore_core/plugin_management/import_plugin.py` will have to be updated to handle stimulus sets. Suggest something like an additional arg indicating that this is a stimulus set, e.g.

`import_plugin('brainscore_vision', 'data', identifier, stimulus_set=True)`

and then in `locate_plugin()`,

```
registry_name = self.plugin_type.strip(
                    's') + '_registry'  if not stimulus_set else `stimulus_set_registry`
```

Reply:
Thank you; I forgot that the identifier is not automatically the directory name

Reply:
resolved in https://github.com/brain-score/core/pull/25